docs/source/api/plugin/landing-pages.md

```diff
@@ -101,6 +101,20 @@ By default, the landing page displays a footer that links to the documentation t
 </td>
 </tr>
 
+<tr>
+<td>
+
+###### `embed`
+
+`boolean`
+</td>
+<td>
+
+By default, the local landing page displays a page with a link to Apollo Sandbox to query your endpoint. If `embed: true` is passed, the landing page will instead render an embedded version of Apollo Sandbox hosted on your endpoint url. From this embedded Sandbox, you can change the endpoint to query against, and use all the features of the Apollo Sandbox on your Apollo Server endpoint. You can also sign in to your Apollo Studio account to access your Operation Collections, Diffs, Checks and more. The default value for `embed` is `false`.
```

Comment:
In general this page could have some links to Sandbox and Explorer docs when the products are mentioned (perhaps in the pre-existing descriptions rather than in the options, or in both)